modules/reference/pages/rpk/rpk-container/rpk-container-status.adoc

```diff
@@ -10,6 +10,29 @@ Get a local container's status.
 rpk container status [flags]
 ----
 
+== Examples
+
+If you're following the guide xref:get-started:quick-start.adoc[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container status` to see more information about your containers:
```

Comment:
```suggestion
If you're following the guide xref:get-started:quick-start.adoc#deploy-redpanda[QuickStart -  Deploy Redpanda to Docker with Three Nodes], you can run `rpk container status` to see more information about your containers:
```

@paulo: I wanted to link straight to the 3 broker steps, but couldn't get this to work xref:get-started:quick-start.adoc?tab=tabs-1-three-brokers. Are we able to link into tabs?